app/views/projects/_list.html.erb

```diff
@@ -0,0 +1,33 @@
+<table class="table table-striped">
```

Comment:
Where is this `list` partial actually used? Is it needed? we already have `app/views/projects/_table.html.erb`

Reply:
I removed it, it somehow appeared during a merge.